sqlparse/filters.py

```diff
@@ -737,6 +741,18 @@ def _create_column_type_length(self, token_queue):
                 tokens=parenthesis_token.tokens
             )
 
+    def _create_column_type_values(self, token_queue):
+        if len(token_queue) <= 0:
+            return None
+        if isinstance(token_queue[0], sql.Parenthesis):
+            parenthesis_token = token_queue.popleft()
+            return sql.ColumnTypeValues(
+                tokens=[sql.Token(
+                    value=token.value.strip('`"\''),
```

Comment:
This may not be correct. Based on the mysql ([enum](http://dev.mysql.com/doc/refman/5.7/en/enum.html) and [set](http://dev.mysql.com/doc/refman/5.7/en/set.html) document, the value is defined as quoted string literal. The enum type like '"abc"' or 'def\'' are valid, i.e. the following statement can create the table successfully: `create table foo (bar enum ('"abc"', 'def'''));`, and the enum strings will be `"abc"`, and `def'`.